package.json

```diff
@@ -13,7 +13,7 @@
     "cross-fetch": "^3.1.5",
     "crypto-browserify": "^3.12.0",
     "scrypt-js": "^3.0.1",
-    "tiny-secp256k1": "^1.1.6"
+    "tiny-secp256k1": "^2.2.1"
```

Comment:
In the past, @ChrisCho-H has downgraded to remove WASM. Please check the commit referenced below. :)

- commit: [[chore: downgrade tiny-secp256k1 to @1.* to remove WASM](https://github.com/hippocrat-protocol/hippocrat-wallet-sdk/commit/ce9b0c5ab7a11408b81d7d370ed2b089a874425d)]

Reply:
Reverted the upgrade